services/api/src/utils/logging.js

```diff
@@ -1,22 +1,173 @@
-function timeStr() {
-  return new Date().toString();
+const tracer = require('./tracer');
+
+const startTime = Symbol('startTime');
+const isLive = process.env.NODE_ENV == 'production';
```

Comment:
feel like `isProd` is a little more understandable no?

Reply:
he yah, was thinking the same the other day.